compose-network/network-node/data/config/api-permission.properties

```diff
@@ -1,62 +1,43 @@
-# NOTE :: api values defines range for example 0 - 1000 then between this range of payer account id are allow to call api.
-# empty value wil be consider as api is not implemented. Also if range end with * will consider as any number of
-# account id can call this api.
-
-# crypto api
+# Crypto
```

Comment:
[These configs live](https://github.com/hashgraph/hedera-services/tree/master/hedera-node/configuration/dev) on the services repo. Since we already ensure the appropriate dev configs are kept up to date for each release, it may be better to source them directly from that repo, so we don't end up with config drift by duplicating here.

Ditto for application.properties, bootstrap.properties, log4j.xml, etc.

Reply:
After talking with Nathan and Neeha it sounds like they will add a GitHub action to keep these in sync from the services repo. I'll mark this as resolved and approve the PR.